README.md

```diff
@@ -1,2 +1,59 @@
-# portfolio-mobile-skeleton
-The project is to build a mobile website using the template chosen in the previous step.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+
+> The project is to build a mobile website using the template chosen in the previous step..
+
+
+## Built With
+- HTML
+- CSS
+
+## Live Demos
+
+[Live Demo Link](https://jbrime15.github.io/portfolio-mobile-skeleton/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Webbrowser and access to internet
+
+### Install
+
+1) Clone the repository to your machine;
+
+```sh
+$ git clone https://github.com/jbrime15/portfolio-mobile-skeleton
+```
+
+2) Navigate to the project folder and execute the "index.html" file.
+
+
+## Author
+
+👤 **JUBRIL IBRAHIM**
+
+- GitHub: [@jbrime15](https://github.com/jbrime15)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL]
- The `MIT` IN in the README file is broken. I would highly recommend fixing this link to make README professional.